Hw5.py

```diff
@@ -0,0 +1,74 @@
+import itertools
+import sys
+
+import mariadb
+import pandas as pd
+import plotly.express as px
+
+
+def __main__():
+    try:
+        conn = mariadb.connect(
+            user="root",
+            password="root",  # pragma: allowlist secret
+            host="localhost",
+            port=3306,
+            database="baseball",
+        )
+    except mariadb.Error as e:
+        print(f"Error connecting to MariaDB Platform: {e}")
+        sys.exit(1)
+
+    # Get Cursor
+    cur = conn.cursor()
+    cur.execute(
+        """
+        Select * from batter_total
+    """
+    )
+
+    df = pd.DataFrame(cur.fetchall())
+    print(df)
+
+    response = ["win_lose"]
+    cat_col = []
+    cont_col = []
+    cat_response = []
+    cont_response = []
+
+    # important loop that determines categorical variables.
+    for i in df:
```

Comment:
since we only have one dataset, why not set categorical columns to dtype category? This would allow for a super easy split